### DISCUSSION TITLE
Error using getRepoSecret

### DISCUSSION THREAD

**Discussion by dzabeu** [octokit/octokit.js#1847] [Q&A]
Hi!

Please, i`m trying to get the repositories'secret , but didnt work to me until now.

This is the code

```js
const secret = await octokit.actions.getRepoSecret({
  owner: 'dzabeu',
  repo: 'daniloteste',
  secret_name: 'ALLOW_REPO_VIEW_ORGANIZATION',
});
```

What is the issue here?

**Comment by gr2m** [comment]
Are you getting an error?

You can not retrieve the value of the secret via the REST API, only GitHub actions have access to the values of the repository secrets

**Reply by dzabeu** [reply]
Hmm i didnt know about i cant get the value! 

But its not necessary, i just want to check if the secret exist or not.

This is the error:

```
RC02XC3X4JG5M:github cx812ng$ node git.js
(node:34093) UnhandledPromiseRejectionWarning: HttpError: Not Found
    at response.text.then.message (/Users/cx812ng/Desktop/github/node_modules/@octokit/request/dist-node/index.js:66:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:34093) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:34093) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Reply by gr2m** [reply]
I can only guess if I don't see the code. A 404 response usually means that you don't have access to the repository you are trying to access. I was trying to reproduce the problem but the request worked as expected. Here is my test case:

https://runkit.com/gr2m/octokit-rest-js-1847/1.0.0

Can you log the error object using `console.log(error)`? It should have additional properties, such as `headers` and `request`.

**Reply by dzabeu** [reply]
the same error!

But i found the issue, i needed to change the owner, was incorrect! Sorry and thanks for your support!
